lib/puppet/provider/service/init.rb

```diff
@@ -144,9 +144,7 @@ def search(name)
       else
         self.debug("Could not find #{name} in #{path}")
       end
-    end
 
-    paths.each do |path|
```

Comment:
Changing the order
from: `a/foo`, `b/foo`, `a/foo.sh`, `b/foo.sh`
to: `a/foo`, `a/foo.sh`, `b/foo`, `b/foo.sh`
seems like the kind of thing that could bite someone. I don't think it's super likely to have both `foo` and `foo.sh` in different directories, but why take the chance?

Reply:
Ok, I deleted the CombineableLoop changes and just moved the excludes out of the `rubocop_todo`.